content/en/real_user_monitoring/funnel_analysis/_index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Funnel Analysis
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "/real_user_monitoring/explorer/"
+  tag: "Documentation"
+  text: "RUM Explorer"
+---
+
+## Overview
+
+Funnel analysis helps you track conversion rates across key workflows to identify and address any bottlenecks in end-to-end user journeys. Specifically, you can:
+
+- See if customers drop off at a certain point due to poor website performance
+- Track how the conversion rate changes over time as new features are built
+- Measure the impact of how adding new steps to a workflow impacts dropoff rate
+
+**Note:** The conversion rate is the number of visitors to your website that completed a desired goal (a conversion) out of the total number of visitors.
+## Build a funnel
+
+To build a funnel, navigate to the Funnels tab within RUM.
+
+{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-funnels-tab.jpg" alt="Navigate to the Funnels tab within RUM" style="width:100%;" >}}
+
+From the funnels page, build your starting view or action and click on the plus icon to proceed in building additional steps. You can also use drag and drop functionality to move steps around.
+
+{{< img src="real_user_monitoring/funnel_analysis/funnel-analysis-building-a-funnel-1.mp4" alt="Filtering network map with search" video=true >}}
+
+### Suggested next steps panel
```

Comment:
```suggestion
### Suggested next steps
```

since it looks like this panel isn't called "Suggested next steps" (looks like the heading of the panel is "Quickly add a step") -- so let's call out the suggestion functionality of this panel without referring to it as the "suggested next steps panel" as that could cause some confusion